web_server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const http = require('http');
+const socketIO = require('socket.io');
+
+const app = express();
+const server = http.Server(app);
+const io = socketIO(server);
+
+server.listen(process.env.PORT || 3000);
+
+// Exposes the folder frontend
+app.use(express.static(__dirname + '/frontend/dist'));
```

Comment:
It's be better to use `path.join()` here since it joins the supplied path segments together with the platform specific seperator.

Reply:
Fixed